File: ejercicios-repaso/Ej-1.c
#include <stdio.h>

int main(void){
    int numeros, suma = 0;

    for(int i = 1; i < 4; i++){
        printf("Ingrese el numero %d: ", i);
        scanf("%d", &numeros);
        suma += numeros;
    }

    char *resultado = (suma == 87) ? "La suma da 87" : "La suma no da 87";

    printf("%s\n", resultado);
    return 0;
}

File: ejercicios-java-propuestos/ejercicios_propuestos/src/ejercicios_propuestos/ej_1.java
package ejercicios_propuestos;
import java.util.Scanner;

public class ej_1 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n1;
		int n2;
		
		System.out.println("Ingrese el numero 1: ");
		n1 = sc.nextInt();
		System.out.println("Ingrese el numero 2: ");
		n2 = sc.nextInt();		
		
		System.out.println("La suma: "+ (n1 + n2));
		System.out.println("La resta: "+ (n1 - n2));
		System.out.println("La multiplicación: "+ (n1 * n2));
		System.out.println("La división: "+ (n1 / n2));
		
		sc.close();
	}
}
